packages/flutter/lib/src/material/material_state.dart

```diff
@@ -321,12 +321,6 @@ abstract class MaterialStateBorderSide extends BorderSide implements MaterialSta
   /// const constructors so that they can be used in const expressions.
   const MaterialStateBorderSide();
 
-  /// Returns a [BorderSide] that's to be used when a Material component is
-  /// in the specified state. Return null to defer to the default value of the
-  /// widget or theme.
-  @override
-  BorderSide? resolve(Set<MaterialState> states);
```

Comment:
Some methods needed to be moved down due to the "constructors should come before other stuff" rule.